kinto_wizard/__main__.py

```diff
@@ -29,6 +29,9 @@ def main():
     subparser.add_argument('--force',
                            help='Load the file using the CLIENT_WINS conflict resolution strategy',
                            action='store_true')
+    subparser.add_argument('--delete',
```

Comment:
@leplatrem Do you want me to use something less generic here to be future proof?

- ``--delete-missing-records``
- ``--delete-records``